upstream-community-operators/hazelcast-enterprise/hazelcast.package.yaml

```diff
@@ -0,0 +1,5 @@
+packageName: hazelcast-enterprise
+channels:
+- name: alpha
+  currentCSV: hazelcast-enterprise-operator.v0.0.1
```

Comment:
This should match the CSV name `hazelcast-operator.v0.0.1`

Reply:
Updated